git-replace, w: add pages (#4496)
tldr-pages/tldr: pages/linux/w.md

# w

> Display who is logged in and their processes.
> More information: <https://www.geeksforgeeks.org/w-command-in-linux-with-examples/>.

> - Show information of all users logged in:

Reviewer: ```suggestion
- Display information of all users logged in:
```

> - Display information of all users logged in:

Reviewer 2: Sorry to be picky, but perhaps this will read better?
```suggestion
- Display information about all users who are currently logged in:
```

Author: Your suggestion looks more readable. Thanks!

`w`

> - Show information of specific user:

Reviewer: ```suggestion
- Display information for a specific user:
```

> - Display information for a specific user:

Reviewer 2: Perhaps this will read better?
```suggestion
- Display information about a specific user:
```

`w {{user}}`

> - Don’t print the header:

Reviewer: ```suggestion
- Display information without include the header:
```

> `w -h`

Reviewer: We prefer long argument format for examples:
```suggestion
`w --no-header`
```

> - Display information without include the header:

Reviewer 2: Perhaps this will read better?
```suggestion
- Display information without including the header:
```

> - Short format,ignore login time, JCPU or PCPU times:

Reviewer: ```suggestion
- Display information without include login, JCPU and PCPU columns:
```

> `w -s`

Reviewer: ```suggestion
`w --short`
```

> - Display information without include login, JCPU and PCPU columns:

Reviewer 2: Perhaps this will read better?
```suggestion
- Display information without including the login, JCPU and PCPU columns:
```
